.travis.yml

```diff
@@ -9,4 +9,4 @@ node_js:
   - "stable"
 
 script:
-  - npm run lint+build+test
+  - npm run build+test+lint
```

Comment:
I never get these tasks. Why not simply run them one after another. If the test fails, you still want to see lint results